docs/src/main/asciidoc/grpc-xds.adoc

```diff
@@ -0,0 +1,83 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+= Using xDS gRPC
+include::_attributes.adoc[]
+:categories: serialization
+:summary: This page explains how to enable xDS gRPC usage in your Quarkus application.
+
+This page explains how to enable xDS gRPC usage in your Quarkus application.
+
+IMPORTANT: This Quarkus xDS gRPC integration currently doesn't support `native` runtime due to the issues
+with sharded grpc-netty library while running native IT tests.
```

Comment:
Either `shaded` or `shared`?
```suggestion
with shaded grpc-netty library while running native IT tests.
```

Reply:
shaded :-D 

But I think we can write a _law_: all libraries using a shaded netty variant are sharding the netty space